core/includes/install.core.inc

```diff
@@ -535,16 +535,19 @@ function install_tasks($install_state) {
   // Determine whether translation import tasks will need to be performed.
   $needs_translations = count($install_state['translations']) > 1 && !empty($install_state['parameters']['langcode']) && $install_state['parameters']['langcode'] != 'en';
 
-  // Display profile task only when there is more than one non-hidden profile.
+  // Display the "Choose profile" task only when there is more than one
+  // non-hidden profiles; hide it if there is only a single exclusive profile.
```

Comment:
There shouldn't be an 's' on the end of 'profiles' here (see original comment you're replacing).